Cargo.toml

```diff
@@ -101,7 +101,7 @@ k8s-openapi = { version = "0.21.1", features = ["v1_28"] }
 pathdiff = "0.2.1"
 kube = "0.88.1"
 rcgen = "0.12.1"
-reqwest = { version = "0.11.24", default-features = false, features = ["rustls-tls"] }
+reqwest = { git = "https://github.com/MathieuDutSik/reqwest", rev = "dff1d229d247c877e8c25da55fde27a4cdbb4ce8", default-features = false, features = ["rustls-tls"] }
```

Comment:
Is this really necessary? As pointed out on the `reqwest` issue, you should be able to just do something like:

```rust
builder.body(serialized_json).header(CONTENT_TYPE, "application/json")
```

Reply:
You are right, that is a far better solution.